pepper-apis/src/main/java/org/broadinstitute/ddp/util/LiquibaseUtil.java

```diff
@@ -31,36 +30,44 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class LiquibaseUtil {
+public class LiquibaseUtil implements  AutoCloseable {
 
-    public static final String PEPPER_APIS_GLOBAL_MIGRATIONS = "src/main/resources/changelog-master.xml";
+    public static final String PEPPER_APIS_GLOBAL_MIGRATIONS = "changelog-master.xml";
 
-    public static final String HOUSEKEEPING_GLOBAL_MIGRATIONS = "src/main/resources/housekeeping-changelog-master.xml";
+    public static final String HOUSEKEEPING_GLOBAL_MIGRATIONS = "housekeeping-changelog-master.xml";
 
     private static final Logger LOG = LoggerFactory.getLogger(LiquibaseUtil.class);
 
-    public static final String AUTH0_TENANT_MIGRATION = "src/main/resources/db-changes/tenant-migration.xml";
+    public static final String AUTH0_TENANT_MIGRATION = "db-changes/tenant-migration.xml";
 
```

Comment:
File paths no longer needed. We are reading these as jar resources. Key change below.